src/dialogs/PreviewUploadOverlay.cpp

```diff
@@ -203,3 +203,13 @@ PreviewUploadOverlay::setPreview(const QString &path)
         setLabels(split[1], mime.name(), data_.size());
         init();
 }
+
+void
+PreviewUploadOverlay::keyPressEvent(QKeyEvent *event)
+{
+        if( event->key() == Qt::Key_Escape )
```

Comment:
This should use `QKeySequence::Cancel` instead, to be platform agnostic. so:

```suggestion
        if( event->matches(QKeySequence::Cancel))
```

Reply:
hmm, fair enough